src/test/groovy/hpo/api/util/Loinc2HpoFactorySpec.groovy

```diff
@@ -18,32 +20,36 @@ class Loinc2HpoFactorySpec extends Specification {
     factory.setLoincEntryPath(new ClassPathResource("LoincTableCore_test.csv").file.absolutePath)
   }
 
-  def "AnnotationMap"() {
-    given:
-      int SIZE = 3
+  def "Annotation map has correct size and keys"() {
 
     when:
-      int size = factory.annotationMap().size()
+      def resultMap = factory.annotationMap()
 
     then:
-      size == SIZE
+      resultMap.size() == expectedSize
+      resultMap.keySet().toString() == expectedKeys
+
+    where:
+    expectedSize | expectedKeys
+    3            | '[38230-9, 738-5, 5778-6]'
+
   }
 
-  def "ReverseAnnotationMap"() {
+  def "ReverseAnnotationMap returns #desc"() {
```

Comment:
take a look at my changes here.. @kingmanzhang .. can you refactor the last test to be more spock like?

Reply:
This is really cool feature! Thanks for showing me this.